features/support/helpers.rb

```diff
@@ -0,0 +1,50 @@
+module Helpers
+  include Spinach::DSL
+  #
+  # Methods in this module provide shorthand, handy, or
+  # semantic equivalents for operations performed
+  # frequently in step.rb files.
+  #
+  # @user is available in the Spinach steps
+  #
+  # PASSWORD = "secretpw"
+  def sign_in_non_admin_user
+    @user = FactoryGirl.create(:user, password: PASSWORD)
+    do_login(@user, PASSWORD)
+  end
+
+  def sign_in_admin_user
+    @user = FactoryGirl.create(:admin, password: PASSWORD)
+    do_login(@user, PASSWORD)
+  end
+
+  def sync_page
+    page.has_css?("nav")
+  end
+
+  def has_flash_msg(severity: , containing: )
+    expect(page.find(".flash .#{severity.to_s}").text).to match(Regexp.new(".*#{containing}.*"))
+  end
+
+  # Currently, poltergeist does not have the support for Capybara's accept_alert{}
+  # this helper provides a workaround
+  def my_accept_alert(&block)
+    if ::Capybara.current_driver == :poltergeist
+      find(".user-id-#{@another_user.id} form.button_to .delete").click
```

Comment:
Best solution is to make this a no-op transform for poltergeist, and only pass the block on to `accept_alert` if the driver supports it.

```
def my_accept_alert(&block)
  if ::Capybara.current_driver.respond_to?(:accept_alert)
    accept_alert do
      block.call
    end
  else
    block.call
  end
end
```